Makefile

```diff
@@ -153,9 +155,9 @@ CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
 	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.1)
 
-KUSTOMIZE = $(shell pwd)/bin/kustomize
+KUSTOMIZE = $(GOBIN)/kustomize
 kustomize: ## Download kustomize locally if necessary.
-	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.2.0)
+	go install sigs.k8s.io/kustomize/kustomize/v4@v4.5.4
```

Comment:
This will not download if kustomize is missing. Inconsistent with the way 'controller-gen' and 'setup-envtest' are downloaded and installed.

Reply:
Changed to previous definition